.github/workflows/SAST.yml

```
@@ -0,0 +1,64 @@
+sast-scan:
```

actionlint (1.7.10)

[error] "jobs" section is missing in workflow

(syntax-check)